data-transformers/transformers/statsTransformer.py

```diff
@@ -69,5 +70,46 @@ def createDict(a):
     return d
 
 # Calls Spark
-query = data.map(createDict)
-query.saveToCassandra(cassandraKeyspace, table, ttl=timedelta(hours=1))
\ No newline at end of file
+query = data.map(createEDDict)
+query.saveToCassandra(cassandraKeyspace, table, ttl=timedelta(hours=1))
+
+
+
+IOData = json.loads(lines[-1].decode())
```

Comment:
Lets store all the data, then we get the maximum in the analyser since it is a cumulative number.

This is because this way we can always check the value of the first data, or visualise the data on a timeline in case it is needed.